metaserver/src/main/scala/scala/meta/languageserver/search/SymbolIndex.scala

```diff
@@ -220,47 +267,81 @@ class SymbolIndex(
     }
   }
 
-  /** Returns a list of fallback symbols that can act instead of given symbol. */
-  private def alternatives(symbol: Symbol): List[Symbol] =
-    symbol match {
+  // TODO(alexey) this is not index specific and could be moved somewhere else (like Symbol-ops)
+  case object alternatives {
```

Comment:
We can turn these into extension methods on Symbol

Reply:
> why case object instead of object?

No good reason for this case. Just a habit, I usually make all objects `case object`s. Is there a general reason not to?

Reply:
When I see `case object` I read "pattern match on this object", so if you don't match on it then I'd leave it non-case

Reply:
OK, I'll change it. (but I think that adding `case` doesn't change anything in respect to pattern matching on an object)

Reply:
The object disappeared in ae723fd208632e4ddb613a920370c010807dc1ee